src/oss-find-squats-lib/Mutators/IMutator.cs

```diff
@@ -35,6 +36,18 @@ public IEnumerable<Mutation> Generate(PackageURL arg)
             bool hasNamespace = arg.HasNamespace();
             foreach (Mutation mutation in Generate(hasNamespace ? arg.Namespace : arg.Name))
             {
+                if (mutation.Mutated.Length == 0)
+                {
+                    // Don't make mutations that are empty. i.e pkg:npm/ isn't valid.
+                    continue;
+                }
+                
+                if (mutation.Mutated.Length == 1 && !char.IsLetterOrDigit(Convert.ToChar(mutation.Mutated)))
```

Comment:
```suggestion
                if (mutation.Mutated.Length == 1 && !char.IsLetterOrDigit(
mutation.Mutated[0]))
```
Do we need to convert or just check the first (and only) character of the string?

Reply:
Done!